internal/commands/vaultsecrets/secrets/displayer.go

```diff
@@ -4,13 +4,15 @@
 package secrets
 
 import (
+	preview_secret_models "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/preview/2023-11-28/models"
 	models "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/stable/2023-06-13/models"
```

Comment:
nit: can we just name this `preview_models` instead of `preview_secret_models` since stable is just `models`